packages/matrix-identity-server/src/cron/index.test.ts

```
@@ -1,96 +1,86 @@
-            resolve()
-          }
-        )
-      })
+            resolve();
+          },
+        );
+      });
     } catch (error) {
-      console.log(error)
-      throw error
+      throw error;
```

Biome (2.4.9)

[error] The catch clause that only rethrows the original error is useless.

(lint/complexity/noUselessCatch)